src/cmd/reconcile.go

```diff
@@ -24,13 +24,13 @@ var reconcileCmd = &cobra.Command{
 		config, err := LoadConfig()
 		cobra.CheckErr(err)
 
+		queue := make(chan opslevel_jq_parser.ServiceRegistration, 1)
+		ctx := common.InitSignalHandler(context.Background(), queue)
 		client := createOpslevelClient()
 		common.SyncCache(client)
 		common.SyncCaches(createOpslevelClient(), resync)
-		queue := make(chan opslevel_jq_parser.ServiceRegistration, 1)
-		common.SetupControllers(config, queue, resync)
+		common.SetupControllers(config, queue, resync, ctx)
 		common.ReconcileServices(client, disableServiceCreation, queue)
-		opslevel_common.Run("Controller")
```

Comment:
Should `opslevel_common.Run("Controller")` be `opslevel_common.Run("Controller")`? Or was this bundled up in the `SetupControllers` update?

Reply:
We're initiating the signal handler on line 28 now so this was removed altogether.